setup/security/authentication/x509/index.md

```diff
@@ -101,13 +102,34 @@ Encoding with any another OID can be done by editing the openssl.conf.
 
 ![Example x509 certificate generated](two_roles_x509.png)
 
-## Setting roleOid via halyard
+## Setting roleOid
 
 ```
 hal config security authn x509 edit --role-oid 1.2.840.10070.8.1
 ```
 
-## Enabling x509 via halyard
+### Configuring SSL to require certs
+If you have SSL enabled, you need to set the Apache Tomcat SSL stack to require a valid certificate 
+chain, as required by the Spring Security integration. 
+
+```
+hal config security api ssl edit --client-auth # Set to WANT or NEED
+```
+
+Set `client-auth` to `WANT` if you still want SSL connections to succeed even if the client doesn’t 
+provide a certificate. This is useful if you enable x509 with another authentication method like
+OAuth, LDAP, SAML - when a certificate is not provided, users can still authenticate with one of
+these methods.
```

Comment:
@stewchen , what does `WANT` add over not invoking this command at all?

Reply:
`WANT` will allow the server to use a certificate if available, whereas when this is unset, the server will not support certificates at all.